Extension/src/LanguageServer/configurations.ts

```diff
@@ -386,6 +391,70 @@ export class CppProperties {
         }
     }
 
+    public nodeAddonIncludesFound(): number {
+        return this.nodeAddonIncludes.length;
+    }
+
+    private async readNodeAddonIncludeLocations(rootPath: string): Promise<void> {
+        let error: Error | undefined;
+        let pdjFound: boolean = false;
+        const package_json: any = await fs.promises.readFile(path.join(rootPath, "package.json"), "utf8")
+            .then(pdj => {pdjFound = true; return JSON.parse(pdj); })
+            .catch(e => (error = e));
+
+        const pathToNode: string = which.sync("node");
```

Comment:
This should be inside the `try {` below (avoids calling which.sync when it's not needed).

Reply:
good point.